### PR TITLE
chore(api): Typescript work and minor fixes

### PR DESCRIPTION
**Pull request by Krisztiaan**

### CODE REVIEW

**Comment by Tobbe** [comment]
I'm familiar enough with this code to review/merge it. @peterp, can you please take a look?

File: packages/api/src/functions/graphql.ts [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-import type { GlobalContext } from 'src/globalContext'
-import type { AuthContextPayload } from 'src/auth'
+import type { GlobalContext } from '../globalContext'
+import type { AuthContextPayload } from '../auth'
+import type { APIGatewayProxyCallback } from 'aws-lambda'
 import { ApolloServer } from 'apollo-server-lambda'
-import { getAuthenticationContext } from 'src/auth'
-import { setContext } from 'src/globalContext'
+import { getAuthenticationContext } from '../auth'
+import { setContext } from '../globalContext'
```
Comment (dthyresson) [inline packages/api/src/functions/graphql.ts:9]: Curious, why is "../" preferred over "src"?

Comment (Krisztiaan) [reply packages/api/src/functions/graphql.ts:9]: See #1381 it'll be addressed there, if fixable.

**Comment by Krisztiaan** [comment]
Likely not relevant anymore.

**Pull request closed without merging**